src/redux/books/books.js

```diff
@@ -0,0 +1,31 @@
+const CREATE_BOOK = 'bookstore/books/create';
+const REMOVE_BOOK = 'bookstore/books/remove';
+const initialState = {
+  books: [],
+};
+// ACTIONS
+export const createBook = (book) => ({
+  type: CREATE_BOOK,
+  payload: book,
+});
+export const removeBook = (id) => ({
+  type: REMOVE_BOOK,
+  payload: id,
+});
+
+// REDUCER
+const booksReducer = (state = initialState, action) => {
+  let newArr = [];
+  switch (action.type) {
+    case CREATE_BOOK:
+      newArr = [...state.books, action.payload];
+      return { ...state, books: newArr };
+    case REMOVE_BOOK:
+      newArr = state.books.filter((book) => book.id !== action.payload);
```

Comment:
- [ ] The redux Store is not storing the books after adding them, when I refresh the page they vanish.

![Capture web_30-9-2021_121631_localhost](https://user-images.githubusercontent.com/61562626/135446142-18679363-fdf8-4cc2-ac1e-4bd86c9ac67a.jpeg)

Reply:
I'm not working with the API yet I'm not sure this is a requirement for this milestone

Reply:
It's not an api just the return of the value. But if you feel its not necessary just puah it for another review